v0.3/handbook/libraries/stdlib.md

```diff
@@ -387,118 +391,979 @@ x = random/number[seed: seconds]</code>
   </tr>
 
   <tr>
     <td>
-      <ul class="arglist">
+      <ul>
         <li><strong>text</strong> - the text to be converted</li>
       </ul>
     </td>
     <td>
       <code>really-funny = "LOL"
-down-a-notch = string/uppercase[text: really-funny]</code>
+        down-a-notch = string/uppercase[text: really-funny]</code>
     </td>
   </tr>
 
   <tr>
-    <td colspan="2">
-      <b>string/index-of</b> - Returns the position of the first occurrence of a specified value in a string
+    <td>
+      <h3>string/index-of</h3> - Returns the position of the first occurrence of a specified value in a string
     </td>
   </tr>
 
   <tr>
     <td>
-      <ul class="arglist">
+      <ul>
         <li><strong>text</strong> - the text to be searched</li>
         <li><strong>substring</strong> - the string to be found in `text`</li>
       </ul>
     </td>
     <td>
       <code>// Eve is in developers, starting at an index of 2
-index = string/index-of[text: "developers", substring: "eve"</code>
+        index = string/index-of[text: "developers", substring: "eve"</code>
     </td>
   </tr>
 
   <tr>
-    <td colspan="2">
-      <b>string/codepoint-length</b> - Returns the length of a string in Unicode code points.
+    <td>
+      <h3>string/codepoint-length</h3> - Returns the length of a string in Unicode code points.
     </td>
   </tr>
 
   <tr>
     <td>
-      <ul class="arglist">
+      <ul>
         <li><strong>text</strong> - the string whose length is found</li>
       </ul>
     </td>
     <td>
       <code>// the code point length of the word "unicode"
-string = "unicode"
-length-in-js = string/codepoint-length[text: string]</code>
+        string = "unicode"
+        length-in-js = string/codepoint-length[text: string]</code>
     </td>
   </tr>
 </table>
 
+## HTML
+
+The HTML library is a low level interface for creating HTML elements and responding to events in the browser.
+
+<table>
+  <tr>
+    <td>
+      <h3>#html/element</h3> - renders a record in the browser as an html element of your choosing
+    </td>
+  </tr>
+
+  <tr>
+    <td>
+      <ul>
+        <li><strong>tagname</strong> - determines the type of html element rendered in the browser; can be any html element given as a string</li>
+        <li><strong>style</strong> - (optional) - specific CSS styles can be defined, but must be entered as a subrecord</li>
+        <li><strong>text</strong> - (optional) - defines text within the html element</li>
+        <li><strong>#</strong> - (optional) - any other tags on the record will be applied to the element as classes</li>
+        <li><strong>class</strong> - (optional) defines one or more classes for the element</li>
+        <li>Other attributes - (optional) - other attribute-value pairs will be applied directly to the element for integration with existing JS libraries or debugging</li>
+      </ul>
+    </td>
+    <td>
+      <code>// commits a div in the browser with the text “Hello world!” and the class “hello”
```

Comment:
These code blocks aren't wrapping and don't have whitespace included, so they look pretty bad. https://github.com/cmontella/docs/blob/e788fcd80d5a2b9873677799b0030d0481707387/v0.3/handbook/libraries/stdlib.md#html